src/client/components/App.js

```diff
@@ -77,15 +77,15 @@ const activeStyle = {color:' #ffffff', background: '#006677'}
 const App = () => (
   <div>
     <ul className="nav nav-tabs">
-      <li><NavLink exact activeStyle={activeStyle} to='/'>Supplier</NavLink></li>
-      <li><NavLink activeStyle={activeStyle} to='/registration'>Supplier Registration</NavLink></li>
-      <li><NavLink activeStyle={activeStyle} to='/address'>Supplier Address</NavLink></li>
-      <li><NavLink activeStyle={activeStyle} to='/contact'>Supplier Contact</NavLink></li>
+      <li><NavLink exact activeStyle={activeStyle} to='/supplier'>Supplier</NavLink></li>
```

Comment:
As this is just a test system does it make sense to do this? Also I'm using react-router v4 which is really different from v3 used in onboarding. I'm not sure I want to  invest lots of time of how to do this as it's just a test UI